src/Exception/RoundingNecessaryException.php

```diff
@@ -11,6 +11,8 @@ class RoundingNecessaryException extends MathException
 {
```

Comment:
Why is `NumberFormatException` pure, but not `RoundingNecessaryException`?

Reply:
Sorry, there is no need to mark exceptions with `@psalm-immutable` at all. I forgot to remove it from there, cleaned up.